src/api/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * This is a mocked API so that we don't have to have any backend data to start with.
+ * But we do write this as a set of API Calls so that it won't be complicated to
+ * re-factor it if we ever need to have a backend.
+ */
+
+import ApiFactory from './api';
+
+import StopData, { IStop } from './stops';
+import RouteData, { IRoute } from './routes/index';
+
+const Stops = new ApiFactory<IStop>(StopData);
+const Routes = new ApiFactory<IRoute>(RouteData);
+
```

Comment:
Are you using a convention for naming consts with capital first letters?

Reply:
Yea I generally tend to use PascalCase for const objects that are returned from a constructor and use CONSTANT_CASE if its a const that is one of the primitives. Do you think its best to use different casing as a standard?

Reply:
I'm not aware of one. It was an honest question. The PascalCase just stood out to me is all.

Reply:
OK cool. Yea, not sure when I started doing it, or where I saw it. But just what I tend to do haha.